mailbox/backup/src/test/java/org/apache/james/mailbox/backup/ZipAssertTest.java

```diff
@@ -49,7 +52,8 @@
     public static final byte[] ENTRY_CONTENT_2 = STRING_ENTRY_CONTENT_2.getBytes(StandardCharsets.UTF_8);
     public static final SizeExtraField EXTRA_FIELD = new SizeExtraField(42);
 
-
+    public static final SimpleImmutableEntry<String, byte[]> ENTRY = new SimpleImmutableEntry<>(ENTRY_NAME, ENTRY_CONTENT);
+    public static final SimpleImmutableEntry<String, byte[]> ENTRY_2 = new SimpleImmutableEntry<>(ENTRY_NAME_2, ENTRY_CONTENT_2);
```

Comment:
you probably don't need to have it public, don't you?

Reply:
Indeed